README.md

```diff
@@ -115,50 +129,50 @@ $ docker run trustlines/tlbc-testnet --help
 Before starting the node, create a Docker network to conveniently allow other containers to easily connect to it:
 
 ```sh
-$ docker network create network-laika
+$ docker network create network-tlbc
 ```
 
 When running the node, you typically want to forward necessary ports to the host so that Parity can find and connect to
 peers. Additionally, you might want to mount some volumes to persist configuration and chain data. For instance, to run
 a non-validator node:
 
 ```sh
-$ mkdir -p trustlines/data-laika trustlines/config trustlines/enode trustlines/shared
-$ docker run -d --name laika-node --network network-laika \
-    -v $(pwd)/trustlines/data-laika:/data \
-    -v $(pwd)/trustlines/config:/config/custom \
-    -v $(pwd)/trustlines/enode:/config/network \
-    -v $(pwd)/trustlines/shared:/shared/ \
+$ mkdir -p tlbc/databases/tlbc tlbc/config tlbc/enode tlbc/shared
+$ docker run -d --name tlbc-node --network network-tlbc \
+    -v $(pwd)/tlbc/databases/tlbc:/data \
+    -v $(pwd)/tlbc/config:/config/custom \
+    -v $(pwd)/tlbc/enode:/config/network \
+    -v $(pwd)/tlbc/shared:/shared/ \
     -p 30300:30300 -p 30300:30300/udp \
```

Comment:
We now use port 30302: https://github.com/trustlines-protocol/blockchain/blob/master/tools/quickstart/quickstart/configs/tlbc/docker-compose.yaml#L19
I guess better be consistent

Reply:
:heavy_check_mark: